hazelcast/src/test/java/com/hazelcast/config/ConfigXmlGeneratorTest.java

```diff
@@ -2266,6 +2266,12 @@ private static HotRestartConfig hotRestartConfig() {
                 .setFsync(true);
     }
 
+    private static DataPersistenceConfig dataPersistenceConfig() {
+        return new DataPersistenceConfig()
+                .setEnabled(true)
+                .setFsync(true);
+    }
+
```

Comment:
Has this method been called? It feels we need to call it from `testCacheAttributes()` and from `testMap()` instead of calling `hotRestart()`.